docs/WritingTests.asciidoc

```diff
@@ -261,6 +261,19 @@ the clone source job or specifying the variable explicitly:
 clone_job.pl --from https://openqa.opensuse.org  --host localhost 24 TESTDEBUG=1 SKIPTO=consoletest-yast2_i
 ----
 
+==== Performance impact
```

Comment:
I find the headline a bit strange. We are here in Writing Tests documentation - and then you throw a performance impact into it. I would expect the paragraph you wrote in Pitfalls actually

Reply:
done, updated. please review again